lib/cogctl/actions/groups/add.ex

```diff
@@ -25,13 +25,13 @@ defmodule Cogctl.Actions.Groups.Add do
   end
 
   defp do_add(_endpoint, _, :undefined) do
-    display_error("Email address is required")
+    display_error("User is required")
```

Comment:
Right now the name of the option is "user" which is why I used that here, but yeah we probably need to look at all of the error messages and option names in general.